### PR TITLE
Update all testing protos

### PR DESCRIPTION
**Pull request by voidzcy**
Updates to testing protos have been made to [grpc/grpc](https://github.com/grpc/grpc/tree/master/src/proto/grpc/testing) directly and they are left out-of-date here. This change copies them back here as grpc-proto should be the source of truth.

### CODE REVIEW

File: grpc/testing/compiler_test.proto [hunk 1]
```diff
@@ -0,0 +1,71 @@
+// Copyright 2016 gRPC authors.
```
Comment (dfawley) [inline grpc/testing/compiler_test.proto:1]: This file may have been omitted on purpose.

I wonder if it wouldn't ultimately be best for grpc/grpc to move this to another proto package and test independently.  Unless the tests for the compiler correctness also are testing other proto messages/etc in this proto package.

File: grpc/testing/echo.proto [hunk 2]
```diff
@@ -0,0 +1,69 @@
+
+// Copyright 2015 gRPC authors.
```
Comment (dfawley) [inline grpc/testing/echo.proto:2]: Similar: the date on this file is very old; it was probably omitted on purpose?

Same for the all of the "new", pre-2019 copyright files?

File: grpc/testing/compiler_test.proto (hunk 1 above)
Comment (voidzcy) [reply grpc/testing/compiler_test.proto]: Excluded.

**Review by dfawley — APPROVED** [review]

**Comment by voidzcy** [comment]
FYI, @donnadionne @ericgribkoff @lidizheng for recent changes made to these protos in grpc/grpc. grpc-proto should be the source of truth and we should always make changes here first and then bring into each language's repo.

**Pull request merged**